issueWork/UI/UX__issue/README.md

```diff
@@ -0,0 +1,27 @@
+# UI / UX issue  
+
+## Changes made:  
+
+> 1. Complete overhall of webpage layout based on previously received feedback. Essentially creating a clean and somewhat minimal style.
+ - ### Laws:  
+   - **Aesthetic-Usability**
+   - **Jakob's Law**
```

Comment:
If this site is supposed to look like other sites, tell me what other site this looks like.

Reply:
This is a universal reference as most simple developer portfolios I have seen looked similar.

Reply:
Can you add specific references for simple developer portfolios that you like here?